clang/test/dpct/autocomplete.c

```diff
@@ -24,6 +24,7 @@
 // DASH-NEXT: --gen-build-script
```

Comment:
Please add lit test for the new added option

Reply:
LIT test for --helper-function-dir line added to autocomplete.c 

E2E test for this feature added here: https://github.com/oneapi-src/SYCLomatic-test/pull/438